packages/path-parser/README.md

```diff
@@ -0,0 +1 @@
+# TODO: README
```

Comment:
Just have even a brief description why does this thing exist

Reply:
I intend to. Just a bit busy with other stuff at the moment...